clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/data/task/jedis/JedisTaskRepository.groovy

```diff
@@ -72,12 +85,11 @@ class JedisTaskRepository implements TaskRepository {
     String taskId = "task:${task.id}"
     jedis {
       def pipe = it.pipelined()
-      pipe.hset(taskId, 'id', task.id)
-      pipe.hset(taskId, 'startTimeMs', task.startTimeMs as String)
-      pipe.sadd(RUNNING_TASK_KEY, id)
+      pipe.hmset(taskId, [id: task.id, startTimeMs: task.startTimeMs as String])
       pipe.expire(taskId, TASK_TTL)
       pipe.sync()
     }
+    jedis { it.sadd(RUNNING_TASK_KEY, id) }
```

Comment:
moved this out of the pipeline - want to experiment with redis cluster which doesn't play nice with multi key operations that may hash to different masters